<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IBDialog</class>
 <widget class="QDialog" name="IBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QISOBurn</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qisoburn.qrc">
    <normaloff>:/images/main.png</normaloff>:/images/main.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>Image file</string>
           </property>
           <property name="buddy">
            <cstring>label</cstring>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="lineFileName"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolBrowse">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Burner device</string>
               </property>
               <property name="buddy">
                <cstring>label_4</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineDevice">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string>/dev/cdrom</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Speed</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboSpeed">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>26</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>26</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="maxVisibleItems">
                <number>5</number>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Layer break</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBreak">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>26</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>26</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="maxVisibleItems">
                <number>5</number>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="checkCompat">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>DVD-ROM Compliant</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkDelete">
           <property name="text">
            <string>Delete image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolRefresh">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolSave">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelDevice">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Burner: [HL-DT-ST][DVDRAM GSA-4163B][A102]
Media:  1Bh, DVD+R, MCC/003</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelStatus">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Current speed: 2.5x</string>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushBurn">
           <property name="text">
            <string>Burn</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushAbort">
           <property name="text">
            <string>Abort</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushClose">
           <property name="text">
            <string>Close</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Output messages</string>
         </property>
         <property name="buddy">
          <cstring>label_5</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainLog">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
         <property name="centerOnScroll">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineFileName</tabstop>
  <tabstop>lineDevice</tabstop>
  <tabstop>comboSpeed</tabstop>
  <tabstop>comboBreak</tabstop>
  <tabstop>checkCompat</tabstop>
  <tabstop>checkDelete</tabstop>
  <tabstop>toolRefresh</tabstop>
  <tabstop>toolSave</tabstop>
  <tabstop>pushBurn</tabstop>
  <tabstop>pushAbort</tabstop>
  <tabstop>pushClose</tabstop>
  <tabstop>plainLog</tabstop>
 </tabstops>
 <resources>
  <include location="../qisoburn.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>IBDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
